Recrutify/Recrutify.Host/CustomAttribute/CustomAuthorizeAttribute.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Newtonsoft.Json;
+
+namespace Recrutify.Host.CustomAttribute
+{
+    public class CustomAuthorizeAttribute : AuthorizeAttribute, IAuthorizationFilter
+    {
+        public void OnAuthorization(AuthorizationFilterContext context)
+        {
+            var user = context.HttpContext.User;
+            var projectid = context.HttpContext.Request.Query.FirstOrDefault(p => p.Key == "projectId").Value;
+            Dictionary<Guid, IEnumerable<string>> userReq = new Dictionary<Guid, IEnumerable<string>>();
+
+            foreach (var role in user.Claims)
+            {
+                if (role.Type == "projectroles")
+                {
+                    var values = JsonConvert.DeserializeObject<ParseJsonToDict>(role.Value);
```

Comment:
you can Pass JsonSerializerOptions() with PropertyNameCaseInsensitive = true to DeserializeObject method and not to put json attribute in class